x/crosschain/genesis.go

```diff
@@ -43,9 +42,14 @@ func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState)
 	}
 
 	// Set all the cross-chain txs
-	for _, elem := range genState.CrossChainTxs {
-		if elem != nil {
-			k.SetCctxAndNonceToCctxAndInboundHashToCctx(ctx, *elem)
+	tss, found := k.GetObserverKeeper().GetTSS(ctx)
+	if found {
+		for _, elem := range genState.CrossChainTxs {
+			if elem != nil {
+				k.SetCctxAndNonceToCctxAndInboundHashToCctx(ctx, *elem, func(ctx sdk.Context) string {
+					return tss.TssPubkey
+				})
+			}
```

Comment:
Trying to retain older behaviour , but I feel we should use 
`elem.GetCurrentOutboundParam().TssPubkey` instead of trying to fetch the current tss.

the cctxs in the genstate would already have been created using the existing tss